client/state/happychat/middleware.js

```diff
@@ -18,6 +25,16 @@ export const requestTranscript = ( connection, { getState, dispatch } ) => {
 	);
 };
 
+export const sendRouteSetEventMessage = ( connection, { getState }, action ) =>{
+	const state = getState();
+	const currentUser = getCurrentUser( state );
+	if ( isHappychatClientConnected( state ) &&
+		isHappychatAcceptingChats( state ) &&
+		isHappychatChatAssigned( state ) ) {
+		connection.sendEvent( `Looking at https://wordpress.com${ action.path }?support_user=${ currentUser.username } ` );
```

Comment:
Could we send this data as `meta` instead? That way we just pass the necessary info and the HUD can construct the link. Eg:

```js
connection.sendEvent( { type: 'ROUTE_SET', path: action.path } );
```
We wouldn't need to send the `username` since the HUD already has that context.

Taking this one step further, could we just send the whole Redux action? 😱 
Then we have the flexibility to add all sorts of events easily, not just ROUTE_SET, and handling the display of each one is just a matter of interpreting it in the HUD.

```js
connection.sendEvent( action );
```